src/envo/envo-edit.owl

```diff
@@ -11256,19 +11265,28 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000485> ObjectSomeValuesFrom(<
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000486> (shoreline)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "USGS:SDTS"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000486> "The line of contact between a body of water and the land."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "USGS:SDTS"^^xsd:string) Annotation(rdfs:comment <https://en.wikipedia.org/wiki/Shore>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000486> "A two-dimensional continuant fiat boundary which is located between a landmass and a water body."^^xsd:string)
```

Comment:
I don't think the GESAMP report is a valid dbxref for this - did it actually help you arrive at this definition?

Reply:
No - I will leave it out then.